src/OpenLoco/src/Ui.cpp

```diff
@@ -112,10 +112,9 @@ namespace OpenLoco::Ui
         return _screenInfo->height;
     }
 
-    // TODO: Rename misleading name.
-    bool dirtyBlocksInitialised()
+    bool isInitialized()
     {
-        return Gfx::getDrawingEngine().isInitialized();
```

Comment:
does anything use this function now?

Reply:
Looks like it has users in Gfx.cpp and Ui.cpp

Reply:
I think he means isInitialized in SoftwareDrawingEngine

Reply:
Yes i mean the software drawing engine function

Reply:
It's not used anywhere, I'll remove it.